### PR TITLE
feat(notification): overrides some of the style to match new guidelines

### PR DESCRIPTION
**Pull request by FelixBlaisThon**
### Proposed Changes

https://coveord.atlassian.net/browse/UITOOL-1191

This Pr overrides some of the notification style from plasma that will ultimately be applied to the admin once I add the Notification system.

I gave background, image size and color and border with respect to gustavo's suggestions https://www.figma.com/file/9eXqWkzwcym7vJeod1fa3X/Plasma-Mantine-(WIP)?node-id=963%3A73730&t=W44SQ0iK7yzZch8H-0

I did it this way since it worked, but if you think there is anything better to get the icon to have the color as presented in the figma file with another cleaner method than this, PLEASE share it in this pr.
### Acceptance Criteria

-   [x] The proposed changes are covered by unit tests
-   [ ] The potential breaking changes are clearly identified
-   [ ] [README.md](https://github.com/coveo/plasma/blob/master/README.md) is adjusted to reflect the proposed changes (if relevant)

### CODE REVIEW

File: packages/website/src/examples/form/button/ButtonWithAsyncLoader.demo.tsx [hunk 1]
```diff
@@ -8,6 +9,15 @@ const promise = async () => {
...
         autoClose: false,
+        icon: <CheckmarkSize24Px height={24} />,
+        styles: (theme) => ({
+            root: {
+                '.__mantine-ref-icon': {
+                    backgroundColor: 'transparent',
+                    color: theme.colors.success[6],
+                },
+            },
+        }),
```
Comment (gdostie) [inline packages/website/src/examples/form/button/ButtonWithAsyncLoader.demo.tsx:20]: I'm sorry to be that guy, but having to write this each time we want to show a notification isn't acceptable IMO 😬 We will have to find another way.

I was think we could just specify `color: 'success'` here, and hide this complexity in the Theme itself

Comment (FelixBlaisThon) [reply packages/website/src/examples/form/button/ButtonWithAsyncLoader.demo.tsx:20]: I did the PR exactly for that purpose, don't apologize :P I will fin a way from the theme

**Comment by gdostie** [comment]
This PR should point to your v6 upgrade PR, otherwise the diff is hard to review 😬

**Comment by gdostie** [comment]
`showNotification` should have some default values, because right now it explodes if you just specify a `message`.

1. Open demo link
2. Click on the button page
3. Click on the first button

**Comment by gdostie** [comment]
The default toast doesn't respect any of the new Notification style designs

image

**Review by gdostie — APPROVED** [review]
💯

**Pull request merged**